resources/views/configureQCMetric.html

```diff
@@ -20,29 +20,40 @@
 
             qcMetricsTable += '<table class="labkey-data-region-legacy labkey-show-borders">' +
                                 '<tr>' +
-                                    '<td class="labkey-column-header">Status</td>' +
                                     '<td class="labkey-column-header">Name</td>' +
                                     '<td class="labkey-column-header">Type</td>' +
+                                    '<td class="labkey-column-header">Metric Usage</td>' +
```

Comment:
Consistent capitalization.

```suggestion
                                    '<td class="labkey-column-header">Metric usage</td>' +
```